src/components/templates/Jazzbar/JazzTab/JazzTab.scss

```diff
@@ -1,6 +1,23 @@
 @import "scss/constants";
 
 .music-bar-container {
+  margin-top: 40px;
+
+  &--tableview {
+    margin-top: 0;
+
+    .music-bar-content {
+      @media (min-width: 950px) {
+        grid-template-columns: 260px 1fr 1fr 260px;
+        grid-template-rows: repeat(auto-fill, 195px);
+        grid-template-areas:
+          ".  video video ."
+          ".  video video  ."
+          "items items items items";
```

Comment:
What is this `items` section used for? I didn't see any `grid-area` referencing it?

Reply:
It is in Jazzbar's Room.scss

![image](https://user-images.githubusercontent.com/63194656/109716594-be779a00-7bad-11eb-954b-5aead5829193.png)

Reply:
Ah.. fair.. that feels.. confusing. But I guess we probably don't have a better way right now.

Perhaps adding a comment in `JazzTab.scss` on this line saying that it's used in the other file would help make this more obvious/clear in future.

Reply:
added in https://github.com/sparkletown/sparkle/pull/1217/commits/3dfa5caf56d2ba6f555b16f76265aa343b5a6771